src/pages/components/tabs.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Tabs
+description: >
+  Tabs organize related content within a single page. This helps users alternate between
+  views without refreshing.
+package: '@zendeskgarden/react-tabs'
+propSheets:
+  - tabs/src/elements/Tabs.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To provide pre-filtered views of a single dataset</li>
+      <li>To organize related content and controls within a page</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>As a secondary navigation bar that spans multiple pages</li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## Examples
+
+### Default
+
+The basic usage of a Tab component.
+
+import TabsDefault from '../../examples/tabs/TabsDefault.tsx';
+import TabsDefaultCode from '!!raw-loader!../../examples/tabs/TabsDefault.tsx';
+
+<CodeExample code={TabsDefaultCode}>
+  <TabsDefault />
+</CodeExample>
+
+### Vertical
+
+Tabs are horizontal by default but can be manually set to stack vertically.
+
+import TabsVertical from '../../examples/tabs/TabsVertical.tsx';
+import TabsVerticalCode from '!!raw-loader!../../examples/tabs/TabsVertical.tsx';
+
+<CodeExample code={TabsVerticalCode}>
+  <div style={{ width: '320px' }}>
+    <TabsVertical />
+  </div>
+</CodeExample>
+
+### Disabled
+
+Individual tabs can be disabled to prevent users from interacting with them.
+
+import TabsDisabled from '../../examples/tabs/TabsDisabled.tsx';
+import TabsDisabledCode from '!!raw-loader!../../examples/tabs/TabsDisabled.tsx';
+
+<CodeExample code={TabsDisabledCode}>
+  <TabsDisabled />
+</CodeExample>
+
+## Best practices
+
+### Be mindful of horizontal tab length
+
+When more than 5 tabs are needed or the tab strings are lengthy, consider a vertical tab layout. Tab
+labels are internationalized and many languages contain more characters per word than English.
+
+Horizontal tabs do not wrap or overflow so anything that exceeds the width of the page will become
+truncated and may hide important actions.
+
+import tabHorizDo from '../../images/components-tabs-horizontal-do.png';
```

Comment:
> @hzhu let's figure out the Abstract issue before merging this one. @rossmoody how did you debug this in GraphiQL?

Yes, let's address this issue before merging. The Abstract API defaults the limit of assets to 50 so `components-tabs-horizontal-do` (along with many others) are not retrieved due to the limit.

We'll need to pass in a `limit` to the [abstractClient.assets.file](https://sdk.goabstract.com/docs/abstract-api/#list-assets-for-a-file) API call. It accepts a number. So an arbitrary high number, or [MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) can be used.

@austin94  @rossmoody

Reply:
Let's follow the [Abstract pagination guidelines](https://sdk.goabstract.com/docs/pagination/). The default limit of 50 seems good to me to limit the length of calls.

We could just look loop through pages in our `abstract-asset` plugin.

Reply:
@austin94 great idea! Thanks for sharing the docs. PR for fix is in https://github.com/zendeskgarden/website/pull/57.